src/components/ExerciseSummary.tsx

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable prettier/prettier */
```

Comment:
Probably either discard this or make this eslint disable global by adding to your .eslintrc.json.